.github/workflows/zowe-cli-plugin.yml

```diff
@@ -0,0 +1,66 @@
+name: zowe-cli-plugin
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    env:
+      OS: ${{ matrix.os }}
+      NODE: ${{ matrix.node-version }}
+      NODE_OPTIONS: --max_old_space_size=4096
+
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository) && !contains(github.event.head_commit.message, '[ci skip]')
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Use NPM v7
+      id: npm7
+      run: npm install -g npm@^7
+
+    - name: Install Node Package Dependencies
+      id: install
+      run: npm ci
+
+    - name: Windows only - Add clidriver to PATH

```

Comment:
More of a question - what happens if the clidriver isn't on the PATH on Windows?

Reply:
A lot of [unit tests](https://github.com/zowe/zowe-cli-db2-plugin/runs/3194530683?check_suite_focus=true) failed with `odbc_binding.node` module not found.
More info on #74